test/rubygems/test_gem_bundler_version_finder.rb

```diff
@@ -111,7 +111,7 @@ def test_filter
       assert_equal %w[1 1.0 1.0.1.1], util_filter_specs(specs).map(&:version).map(&:to_s)
     end
     bvf.stub(:bundler_version, v("2.a")) do
-      assert_equal %w[2 2.a 2.0 2.1.1], util_filter_specs(specs).map(&:version).map(&:to_s)
+      assert_equal %w[2.a 2 2.0 2.1.1], util_filter_specs(specs).map(&:version).map(&:to_s)
```

Comment:
I had to change this existing expectation. The new expectation makes more sense to me. If we have a prerelease in the `BUNDLED WITH` section, we still want that activated even if it is a prerelease... right? :thinking:

Reply:
hmmmm good question! I'm not sure how we want to handle autoswitching if the locked version is a prerelease. probably using the exact version is the safest, so I think this is a good place to start.